Dockerfile

```diff
@@ -23,10 +23,11 @@ USER root
 
 # Upgrade NodeJS > 12.0
```

Comment:
This line would deserve an update too.

Reply:
I guess, but I don't rightly know what the current constraints on the NodeJS version are.

AFAIK any LTS NodeJS > 12.0 is still good, since the image works with NodeJS 14. So the comment might actually be correct right now?

Reply:
No better formulation was found.